packages/govtnz-ds-website/src/components/LogoLockUp.tsx

```diff
@@ -31,7 +31,12 @@ const LogoLockUp = ({ tabIndex, ariaHidden }: Props) => {
         </svg>
       </div>
       <div className="header__logo header__logo--hide@medium header__logo--hide-print">
-        <Icon className="header__icon" role="img" id={iconNZGShortLogo.id} />
+        <Icon
+          className="header__icon"
+          ariaHidden={true}
```

Comment:
just fyi you can write valueless attributes if you're passing in boolean true
```suggestion
          ariaHidden
```